### DISCUSSION TITLE
listfinancialevents overclocking

### DISCUSSION THREAD

**Discussion by lemon4acid** [amzn/selling-partner-api-models#4401] [Q&A]
listfinancialevents has been overclocking these days. What is the reason?

**Comment by tspicer** [comment]
We are seeing issues with the Finance API returning empty events since 12/8/24

**Comment by weilinggu** [comment]
Hi @lemon4acid We are actively working on mitigating the issue. Please monitor the [API health dashboard](https://sellercentral.amazon.com/sp-api-status) to get the latest status of the Finance API.

Thanks
Weiling
Amazon Selling Partner Services

**Reply by lemon4acid** [reply]
We were good at pulling sales data before, but around December 4th, the sales data interface was consistently overclocked, while all other interfaces were good. After checking for a few days, we found that Amazon notified us that we needed to resubmit the file to verify the store's identity information. We submitted the information, but after Amazon approved it, we tried to call the sales data again and it still kept overclocking.
Where can we see the API health dashboard you mentioned above? Could you please let me know? Thank you！

**Reply by weilinggu** [reply]
@lemon4acid, 

Can you see the link here? https://sellercentral.amazon.com/sp-api-status

If your issue started happening on Dec 4th (or if you are not in NA region), it might be worth [opening a support case](https://developer.amazonservices.com/support) to verify whether if there are other issues outside of the service issue mentioned in the health dashboard.

Thanks
Weiling
Amazon Selling Partner Services

**Reply by lemon4acid** [reply]
@weilinggu ,
Open link https://sellercentral.amazon.com/sp-api-status  
There seems to be some issues with the sales data. "Service is reported to be experiencing high rate of errors. We are currently investigating the issue."
Do we need to wait for Amazon to handle this?
![截图](https://github.com/user-attachments/assets/8047ade2-a72b-447f-b601-182c3f7c0456)

**Reply by lemon4acid** [reply]
@weilinggu ，
May I ask how to obtain authorization for Amazon SP-API if the merchant cancels the authorization process? Can a callback method be set for revoking authorization?

**Reply by chris1out** [reply]
We are still seeing this even after the health dashboard has been updated to green. Many accounts I manage constantly getting either invalid input or quota exceeded errors.
